code/controllers/subsystem/ticker.dm

```diff
@@ -428,6 +428,40 @@ var/datum/subsystem/ticker/ticker
 				total_antagonists.Add(temprole) //If the role doesnt exist in the list, create it and add the mob
 				total_antagonists[temprole] += ": [Mind.name]([Mind.key])"
 
+	//Shows who completed all the crew objectives
+	var/list/successfulcrew = list()
+	for(var/A in player_list)
+		var/mob/M = A
+		if(M.mind && M.mind.objectives)
+			if(locate(/datum/objective/crew) in M.mind.objectives)
+				var/youdidit = TRUE
+				for(var/datum/objective/crew/C in M.mind.objectives)
+					if(!C.check_completion())
+						youdidit = FALSE
+				if(youdidit)
+					successfulcrew += M.name
+	world << "<font color='green'><b>The following crewmembers managed to complete all their crew objectives: [jointext(successfulcrew, ", ")]</b></font>"
```

Comment:
A massive paragraph in solid green is going to look awful.
Could we change that it's just black bold text and use  <span class='greenannounce'></span> for all the successful crewmembers please?